### DISCUSSION TITLE
Ajout d'une possibilité dans le witch event

### DISCUSSION THREAD

**Discussion by Royal-91** [Crownicles/Crownicles#1661] [Nouvelles idées]
On pourrait pour le witch event avoir la possibilité d'ajouter une de nos potions qui est déjà dans notre inventaire.
Ca permettrait que les potions qu'on stock dans notre inventaire aient une utilité

**Comment by KirIcare** [comment]
C'est pas mal mais je trouve qu'on a trop peu d'inventaire pour faire ça genre les potions je les stock car elles donnent de la vie ou passe du temps, si jamais j'ai le witch avec ça, je les mettraient pas dans le mini event x)

**Comment by Royal-91** [comment]
C'est sur après perso j'essaye de stocker les popo de soin mais 80% du temps j'ai des potions de merde dans mon inventaire

**Reply by QQtin3** [reply]
Je trouve l'idée ultra intéressante personnellement, comme ça je me débarrasse enfin de mes popos useless :v
Nan plus sérieusement c'est vrai que ça peut être cool, à voir ce que ça change pour chaque potion tho

**Reply by KirIcare** [reply]
Moi aussi ça m'intéresse, juste est ce que draft sera open pour faire une feature qui serais peu utilisé ? Je veux dire il faut 1) avoir le witch 2) avoir des popos inutiles (ce qui n'est pas mon cas actuellement par exemple) 3) vouloir faire l'ajout de popo
Il faudrait penser au système de récompense comme à dit QQ pour qu'il soit autant ou plus intéressant que les autres choix
